src/components/NavigationBar/SideMenuLink.svelte

```diff
@@ -2,6 +2,9 @@
 	import type { Menu } from '$models/menu';
 	import ArrowUpRightIcon from 'carbon-icons-svelte/lib/ArrowUpRight.svelte';
 	import { twMerge } from 'tailwind-merge';
+	import { getContext } from 'svelte';
+	const closeSideNav: () => void = getContext('closeSideNav');
```

Comment:
Better move this variable below `export let`. Props is the most important information when anyone want to use components, so we better organize code that way.